packages/sdk-patcher/src/rules/sdk-patcher-rule.ts

```
@@ -8,7 +8,12 @@ export interface SdkPatcherRuleOperationArgs {
   artifacts: Artifacts;
 }
 
-export interface SdkPatcherRule {
+interface SdkPatcherRuleObject {
   install(args: SdkPatcherRuleOperationArgs): void | Artifacts | Promise<void | Artifacts>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)